target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-lbr20.dts

```diff
@@ -0,0 +1,515 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Netgear LBR20";
+	compatible = "netgear,lbr20";
+
+	chosen {
+		bootargs-append = "ubi.mtd=ubi root=/dev/ubiblock0_0";
+	};
+
+	aliases {
+		led-boot = &led_green_status;
+		led-failsafe = &led_green_status;
+		led-running = &led_green_status;
+		led-upgrade = &led_red_status;
+		label-mac-device = &gmac;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&tlmm 49 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_green_status: led-0 {
```

Comment:
`led-status-green` for node name, same logic for other LED-s.

Reply:
I can change order , but names schould be with underscores...
Please see , if this is what you want in corrected commit.

Reply:
I think we have a misunderstading, `led_green_status` aka the label is fine, but instead of just using `led-0` it should be `led-status-green`.

Same logic goes for other LED-s as well

Reply:
Ok , changed names for all leds , and also added boot function to ..backlight_white ( it was possible , whan label was set).